packages/commonwealth/client/scripts/views/pages/view_thread/linked_proposals_card.tsx

```diff
@@ -25,8 +22,8 @@ type ThreadLinkProps = {
 };
 
 const getThreadLink = ({ threadChain, identifier }: ThreadLinkProps) => {
-  const slug = chainEntityTypeToProposalSlug();
-
+  // XXX 7/3/2024: proposal links only supported for cosmos
```

Comment:
does this mean we only render it in cosmos communities?

Reply:
It means only Cosmos communities can have linked onchain proposals (natively, other communities can use web links instead).